session_4/project/README.md

```diff
@@ -0,0 +1,24 @@
+# README
```

Comment:
Please add screenshots of the features you have implemented and describe them in the README like https://github.com/VikramSGIT/IRIS-RoR-Bootcamp-2020/tree/session_4_vikram/session_4/blogspot